pkg/machine/machine_windows.go

```diff
@@ -18,3 +20,24 @@ func GetProcessState(pid int) (active bool, exitCode int) {
 	syscall.GetExitCodeProcess(handle, &code)
 	return code == 259, int(code)
 }
+
+func PipeAvailable(pipeName string) bool {
```

Comment:
Is this function really doing what it's supposed to?

It returns `true` if the path does not exist (and `false` if it exists) but the name/callers suggest differently.

Reply:
Thanks!  Yes, that would help me brain parse the code :)

Reply:
Renamed to `PipeNameAvailable`. Was thinking about `PipeAddress`, because it is more common (at least from my experience) for everything network/connection like, but MS uses Name it their docs https://learn.microsoft.com/en-us/windows/win32/ipc/pipe-names 

`PipeNameAvailable` doesn't look like a masterpiece of naming, but it at least makes it clear what is actually checked by the function.